### ISSUE TITLE
[Feature Request] - Form widget: add  "rating" field type

### ISSUE THREAD

**Issue by admcoding** [elementor/elementor#10713]
## Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by xymantec** [comment]
I agree this feature should be there, we need the star rating as part of the form widgets fields so we can capture ratings via native elementor forms. Right now the work around is frankly and a waste of time. The best solution would be to get this rating update so we can capture customer reviews at which point we can process the results via API.
